nix/status-go/mobile/build.nix

```
@@ -46,6 +46,10 @@ in buildGoPackage rec {
   # TODO: try removing when go is upgraded to 1.22
   GODEBUG = "netdns=cgo+2";
 
+  # Sentry for status-go
+  SENTRY_CONTEXT_NAME = "status-mobile";
+  SENTRY_CONTEXT_VERSION = fileContents ../../../VERSION;
```

Unnecessary. Just use the version we already have.
```suggestion
  SENTRY_CONTEXT_VERSION = version;
```